cos/static/scripts/cookie-banner.js

```diff
@@ -0,0 +1,21 @@
+// Cookie Banner
+$(document).ready(function () {
+    var cookieBannerElem = $('#cookieBanner');
+    var cookieAcceptedKey = 'cos_cookieconsent';
+    var daysToExpire = 30;
+
+    var $cookieBanner = cookieBannerElem.on('closed.bs.alert', function() {
+        var d = new Date();
+        d.setTime(d.getTime() + (daysToExpire*24*60*60*1000));
+        var expires = "expires="+ d.toUTCString();
+        document.cookie = cookieAcceptedKey + "=;" + expires + ";path=/";
```

Comment:
Does a cookie not have to have a value?

Reply:
I didn't think so, but couldn't hurt to set one cause that's definitely clearer, and matches what the OSF is doing. I'll change it to 1!